controller/src/beerocks/master/tasks/association_handling_task.cpp

```diff
@@ -28,7 +28,6 @@ using namespace son;
 #define BEACON_MEASURE_REQ_TIME_SPAN 3000
 #define BEACON_MEASURE_MAX_ATTEMPTS 3
 #define REQUEST_RSSI_MEASUREMENT_DELAY 5000
-#define GET_CLIENT_IP_TIMEOUT_SEC 3
```

Comment:
I think it makes more sense to actually do `git revert <commit>` then `git commit --amend` to update with the (great!) explanation you added in the commit message. This way, the commit headline will include the hash of the reverted commit so its easier for searching in the git log.